libs/shared/providers/src/intl/hooks.ts

```diff
@@ -27,12 +27,17 @@ export const useFormat = () => {
       decimals = 18,
       zeroPlaceholder = '0.00',
       options?: FormatNumberOptions,
+      exchangeRate?: number,
```

Comment:
don't embed business logic here, the responsibility of this function is to format an input, not execute operations on it. If you need to convert the input, it must be done before calling this formatter

Reply:
That is a bummer. This can make life much easier in some situations.